app/shared/src/cards/__tests__/SimpleCard.test.js

```diff
@@ -1,36 +1,39 @@
 // @flow
 
 import * as React from 'react';
-import { View } from 'react-native';
 import { PlaygroundRenderer } from '@kiwicom/mobile-playground';
 import { Translation } from '@kiwicom/mobile-localization';
 
 import SimpleCard from '../SimpleCard';
 
 function onPressVoidCallback() {}
 
+function SimpleCardWrapper(props) {
+  return <React.Fragment>{props.children}</React.Fragment>;
+}
+
 it('renders as expected', () => {
   PlaygroundRenderer.render(
-    <View>
-      <SimpleCard key="A">
+    <SimpleCardWrapper>
```

Comment:
Because we now filter on the name of the top level component exposed to `render`
`.filter(component => component.type.name === this.props.name)`
If we expose `React.Fragment` as the top level component, and another component also use `React.Fragment` both components would show. 

So for this reason, I added a `SimpleCardWrapper` to ensure a unique name